commands/addons/destroy.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+
+const cli = require('heroku-cli-util')
+const co = require('co')
+const resolve = require('../../lib/resolve')
+const util = require('../../lib/util')
+const _ = require('lodash')
+
+function * run (context, heroku) {
+  let force = context.flags.force || process.env.HEROKU_FORCE === '1'
+  if (context.args.length === 0) {
+    throw new Error('Missing add-on name')
+  }
+
+  function showRelease (app, attachments) {
+    let names = attachments.map((att) => cli.color.configVar(att.name)).join(', ')
```

Comment:
Yea, I think it's probably easier to omit it rather than trying to determine the diff via API, somehow.